modules/fscloud/main.tf

```diff
@@ -66,7 +66,10 @@ locals {
     "hs-crypto" : {
       "enforcement_mode" : "report"
     },
-    "containers-kubernetes" : {
+    "containers-kubernetes-management" : {
```

Comment:
Should we keep `containers-kubernetes` as well along with management and cluster?

Reply:
I don't think so, this would result in duplicate rules